src/main/java/com/deloitte/elrr/elrrconsolidate/consumer/KafkaConsumerConfig.java

```diff
@@ -29,9 +31,10 @@ public class KafkaConsumerConfig {
      */
     @Bean
     public ConsumerFactory<String, String> consumerFactory() {
+        log.info("Start building Kafka Consumer factory");
         Map<String, Object> props = new HashMap<>();
         props.put(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, brokerUrl);
-        props.put(ConsumerConfig.GROUP_ID_CONFIG, "GROUP_ID");
+        props.put(ConsumerConfig.GROUP_ID_CONFIG, "test-group");
```

Comment:
i would say this probably needs to be not hardcoded, and should be another prop maybe? I think this prop already exists.